WidgetExtension/LightChart/ChartType.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ChartType.swift
+//  
+//
+//  Created by Alexey Pichukov on 19.08.2020.
+//
```

Comment:
What's the reason you included this directly instead of using SwiftPM for the forked version?

Reply:
I wasn't able to fork using the Mastodon Org, but in general it's better to use a fork, so let me for it to my own org and then we can always move it over.